test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -514,23 +534,38 @@ public void flush()
 
     public void flush(String keyspace)
     {
-        ColumnFamilyStore store = getCurrentColumnFamilyStore(keyspace);
+        flush(keyspace, currentTable());
+    }
+
+    public void flush(String keyspace, String table)
+    {
+        ColumnFamilyStore store = Keyspace.open(keyspace).getColumnFamilyStore(table);
```

Comment:
Suggestion: maybe use the method `getColumnFamilyStore`, e.g.
```
ColumnFamilyStore store = getColumnFamilyStore(keyspace, table);
```

Reply:
Agreed, I'm not sure why it ended up with this. Probably incremental changes.

I've pushed a commit with this change on it.